cmd/lncli/commands.go

```diff
@@ -1977,6 +1977,19 @@ var listChainTxnsCommand = cli.Command{
 				"transactions until the chain tip, including " +
 				"unconfirmed, set this value to -1",
 		},
+		cli.UintFlag{
+			Name: "index_offset",
+			Usage: "The index of a transaction that will be " +
+				"used in a query to determine which " +
+				"transaction should be returned in the " +
+				"response, where the index_offset is " +
+				"excluded",
+		},
+		cli.IntFlag{
+			Name: "max_transactions",
+			Usage: "the max number of transactions to return. " +
+				"Set to -1 to get all transactions",
```

Comment:
This flag should have the default value of `-1`, otherwise by default the command wouldn't return any results anymore. Then the description should probably be updated to `(optional) the maximum  number of transactions to return; leave at default of -1 to return all transactions`.